src/main/kotlin/solve/main/MainView.kt

```diff
@@ -47,9 +70,105 @@ class MainView : View() {
     private val rightSidePanelViews =
         createSidePanelsViews(rightSidePanelTabs, SidePanelLocation.Right)
 
+    private val importFAB = mfxButton {
+        VBox.setMargin(this, Insets(5.0, 8.0,10.0,8.0))
+
+        val circle = Circle(this.layoutX + 28.0, this.layoutY + 28.0,28.0)
+        clip = circle
+        graphic = ImageView(importIcon)
+        prefHeight = 56.0
+        prefWidth = 56.0
+
+        style = "-fx-background-color: #${Style.secondaryColor}; -fx-background-radius: 28;"
+
+        action{
+            importAction()
+        }
+    }
+
+    private val pluginsButton = mfxButton("Plugins") {
+        clip = Style.circleForRipple(this)
+        contentDisplay = ContentDisplay.TOP
+        setPrefSize(Style.navigationRailSize, Style.navigationRailSize)
+        styleClass.add("mfxButton")
+        prefHeight = 70.0
+        prefWidth = 70.0
```

Comment:
Why you set size only for this button? It actually affects background circle
![image](https://user-images.githubusercontent.com/55746901/233857438-86c298d0-649a-454d-9e04-61775c077854.png)
![image](https://user-images.githubusercontent.com/55746901/233857448-24f81885-8810-4521-8a5c-610cb3d74686.png)